opencti-platform/opencti-front/src/utils/sessionStorage/setStackDataInSessionStorage/setStackDataInSessionStorage.ts

```diff
@@ -0,0 +1,17 @@
+type SetDataOnSessionStorageParams = {
+  valueToStore: unknown;
+  key: string;
+  stackValue: number;
+};
+const setStackDataInSessionStorage = ({ valueToStore, key, stackValue }: SetDataOnSessionStorageParams): void => {
```

Comment:
I think we usually go for inlined parameters and not an object.
I suggest also this order and naming. It's more standard to use key-value wording I think.

```suggestion
const setStackDataInSessionStorage = (key: string, value: unknown, stackLimit: number): void => {
```